include/rtc/include.hpp

```diff
@@ -56,6 +56,7 @@ using std::byte;
 using std::string;
 using std::string_view;
 using binary = std::vector<byte>;
+using binary_ptr = std::shared_ptr<binary>;
```

Comment:
Just a question: what is the difference of meaning between `message_ptr` and `binary_ptr`?

Reply:
`binary_ptr` is shared pointer to `vector<byte>`, message_ptr is shared pointer to `Message` (subclass of `vector<byte>`), I added `binary_ptr` because I didn't need subclass of `vector<byte> (`Message`).